.github/workflows/testing.yaml

```diff
@@ -0,0 +1,22 @@
+name: Testing
+
+on:
+  - pull_request
+
+jobs:
+  Test:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-tags: true
+          fetch-depth: 0
+      - name: Build Image
+        run: |
+          make autok3s
+          docker build . -t cnrancher/autok3s:dev
```

Comment:
For the version parsing error, I am doing `make autok3s` and `docker build -t cnrancher/autok3s:dev`, which are the steps you are trying to run on Github action.

Reply:
Any version string is ok, because we don't need to push the image to a registry. But I don't know why I cannot see the error. [This is](https://github.com/LinuxSuRen/autok3s/pull/1) the PR and GitHub action running record in my forked repo.

Reply:
No logs show in the workflow, this is also the issue of this PR. I don't think you run the right `docker-compose.yml` in workflow runs.

Reply:
Yes, I just notice that. Fixed, see https://github.com/LinuxSuRen/autok3s/actions/runs/7109535965/job/19354609320?pr=2

Reply:
I found the error 
![image](https://github.com/cnrancher/autok3s/assets/1450685/b845d627-75b8-433c-9609-eb24688381f4)

Reply:
But I don't understand how to fix the version issue.

Reply:
You can either change the version parsing logic(hard one) or give it a tmp tag(easy one).
Or you can modify the `hack/lib/version.sh` to change the version logic.